### ISSUE TITLE
Implement new format for community call

### ISSUE THREAD

**Issue by patcon** [edgi-govdata-archiving/overview#163]
## To Do

- [x] research/refresh on resources for community calls
- [x] host discussion about thoughts and feelings on community calls
- ~Integrate and/or address @lightandluck's [feedback for community call format](https://archivers.slack.com/archives/C4HNJTTL6/p1498793490166698)~
- ~Update [guidelines](https://github.com/edgi-govdata-archiving/overview/blob/master/guidelines/community_calls.md) with new format~
- ~Switch to new community calls format!~

(@dcwalk edit: added a couple todos from Thurs Aug 10 call)

**Comment by lightandluck** [comment]
*Copy/pasted with small edits*

1) Set aside 5min for demos or project overviews. _Demos don't have to happen at every meeting. Maybe when something new comes out or if we have new people at meetings._ A lot of terms and projects are bandied around like Climate Central, Data Together, etc. which not everyone knows about. At least not me! It would be nice to have general introduction/overview of the different projects going on. Great for newcomers and people like me who don't have that high-level overview of everything happening in EDGI.

2) Signup sheet at the beginning of meetings in chat for participants who have updates they want to share, so others who just want to listen in or don't have updates don't have to be put on the spot. Also, project updates usually include the speaker's updates and they don't have to repeat themselves in the go-around.

3) EDGI bios - It would be great to get to know each other a little more and learn about why **we choose** to be here together.  Ask for 1 volunteer at beginning of meeting for quick 2min introduction. 
Answer 2 questions: 
 1- How did you find/get involved with EDGI? 
 2- What keeps you motivated/why do you do the work that you do for EDGI? 

I don't think anybody here is in it for the money or the fame :money_mouth_face:, so it would be nice to hear about and remind ourselves why we're really here.

**Comment by patcon** [comment]
Thanks!

Some resources I was hoping to catch up on:

- [How to run a Community Call](http://www.zythepsary.com/learning-materials/how-to-run-a-community-call/)
- [Mozilla Learning Community Calls](https://learning.mozilla.org/en-US/community/community-call)
- [Mozzilla Science Community Call](https://science.mozilla.org/programs/events/community-call-aug-10-11-et)

**Comment by dcwalk** [comment]
@patcon -- do you have a sense on when you are thinking of moving on this? I think starting to trial and get feedback on different formats could be great! (Especially with the support of other regular project calls and the archiving call)

(edit: typos :()

**Comment by patcon** [comment]
I'm planning to cap today's Community Call by leading a final discussion to workshop the community call format -- I feel I need a little more feedback on feelings before putting forward a proposal, as it's potentially quite a big change.

For posterity, here is the deeplink to the YouTube video from last month where we had the first convo: https://www.youtube.com/watch?v=wEvcC46pl-0#t=25m35s

Here's the agenda for tonight, with a link to a brainstorming doc I hope we can build on tonight:
https://hackmd.io/IwBgnAhgpgTMAcBaAJskBmRAWAbAdhEQCMBWAY0JOjBhGSyzKKyA?view

**Comment by dcwalk** [comment]
Just to flag that here is where we codified the process: https://github.com/edgi-govdata-archiving/overview/blob/master/guidelines/community_calls.md 

So that should be updated with the new format :)

**Comment by dcwalk** [comment]
Hey @patcon how are things going with this? any next steps I can assist with?

**Comment by dcwalk** [comment]
Based on the [September 11 Archiving call](https://hackmd.io/GwFgJgRgrGCMCGBaAxgU1WRIDMFP1gA4B2FeCEAgTm3mAAZgg===?view), it sounds like there are still things that need to be formalized and this issue needs to be updated. 

@patcon does that match your understanding?

**Comment by dcwalk** [comment]
Hey @patcon just hoping to get a brain dump on this, as I know you've mentioned shifting your focus. Can you maybe summarize what you see as left before we can mark this issue as complete?

**Comment by dcwalk** [comment]
Circling back @patcon -- Can you maybe summarize what you see as left before we can mark this issue as complete?

**Comment by patcon** [comment]
Hey @dcwalk, this deserves a fuller response, but the short of it was that I was starting to feel that community call (though perhaps valued) was not quite as important as onboarding. And then we had a few weeks in a row after the holidays where no one showed up, so I think I lost a bit of energy there. (Though to be fair, I could have rallied people more on seeing that other weren't showing up, so I will own up to that.)

I think a fair wrap-up for this would be to do **put out an ~open call~ ask, searching for someone else to take on hosting of the Community Call.**

If I were to find time to put toward something like this in the near future, it would be more geared toward onboarding.

**Comment by weatherpattern** [comment]
Maybe this could be a monthly thing and open to new comers and long time volunteers. 

The Web Monitoring Team has started infrequent once every two months or so, social hangoutlcalls. We've had 3 so far. They are fun way to get to know the team. It is more relax and a really good break the pressure of getting through an agenda on the weekly call. Having them infrequently make them more of a special occasion that people make time for in their busy lives.

**Comment by patcon** [comment]
That's a good thought ray. I did like them a ton! Currently organizing a civic hackathon in Toronto, so I need time to plot timeline for that. (I feel it needs some prep time that I won't have until after Mar 3)

**Comment by dcwalk** [comment]
Hey all, I think we might want to close this issue as it doesn't appear to be an active conversation and an area that we are interested in implementing (e.g., updating the guidelines). 

I think anythings like the call/ask will happen in another venue and we can create new issues at that time
